packages/components/storybook/test/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import initStoryshots from '@storybook/addon-storyshots';
+import path from 'path';
+
+/**
+ * The list of components that should be skipped because they
+ * don't work with the default Storyshots setup.
+ *
+ * @type {string[]}
+ */
+const skippedComponents = [
+	'ClipboardButton',
```

Comment:
This raises a few questions.
- do we want to mock anything on the case by case basis if something doesn't work out of the box?
- what's the goal of snapshot testing in this context? 
- are we sure that there isn't an issue with the current implementation of the `ClipboardButton` components?

I don't feel strongly about having 100% coverage through stories. I find StoryShots a nice tool that helps us to prevent regressions but not necessarily something which is mandatory for each component.

Reply:
>do we want to mock anything on the case by case basis if something doesn't work out of the box?

It's not on a case by case basis, refs can be mocked generically, but yeah, why not?

>what's the goal of snapshot testing in this context?

To make sure the classes and markup of the button don't change in a breaking way.

>are we sure that there isn't an issue with the current implementation of the ClipboardButton components?

Refs are just not supported and we need to mock them.

Reply:
> To make sure the classes and markup of the button don't change in a breaking way.

This is where I'm a little suspect at the value of snapshot testing. I'm assuming that any logic in a component that creates a meaningful markup change/variation (e.g. classnames, a11y attributes etc) would be tested by regular unit tests (is that correct?) and if that assumption is true then it doesn't feel like snapshot tests would provide much value over a `git diff` of the component for a developer to be aware of whether hard-coded markup and attributes have changed?

Is there something else I'm missing? 🤔

Reply:
>I'm assuming that any logic in a component that creates a meaningful markup change/variation (e.g. classnames, a11y attributes etc) would be tested by regular unit tests (is that correct?)

That is rarely the case, and snapshot testing makes this easier anyways.

Reply:
Thanks for clarifying @epiqueras!